.eslintrc.json

```diff
@@ -5,7 +5,8 @@
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
     "no-unused-vars": "off",
-    "unused-imports/no-unused-imports": "error"
+    "unused-imports/no-unused-imports": "error",
+    "react/no-unescaped-entities": "off"
```

Comment:
remove this line. You'll need to replace all occurences of ' in the texts by &apos; to avoid errors